### PR TITLE
Remove the login click for native (bug 1103103)

### PR DESCRIPTION
**Pull request by muffinresearch**
This removes the login click that was previously required for persona when using nativeFxA.

I've also killed the basic test as it didn't really do anything useful.

r? @kumar303 @washort

### CODE REVIEW

File: public/js/views/page.js [hunk 1]
```diff
@@ -72,12 +72,12 @@ define([
...
         if (utils.useOAuthFxA()) {
-          // FxA
+          // FxA Web Flow
           utils.fxaLogin();
         } else {
-          // Persona
-          logger.log('Displaying login view.');
-          app.router.showLogin();
+          // Native FxA
+          app.throbber.render(this.gettext('Connecting to FxA'));
```
Comment (kumar303) [inline public/js/views/page.js]: is this a user message? If so I think it should say something more like "Connecting to login service" or maybe "Connecting to Firefox Accounts"

File: tests/ui/test-basic.js [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-    // Run the tests.
-    casper.waitForSelector('#app', function then() {
-      test.assertVisible('#app');
-    });
-
-    casper.run(function() {
-      test.done();
-    });
-  },
-});
```
Comment (kumar303) [inline tests/ui/test-basic.js]: heh, yeah

**Comment by kumar303** [comment]
if FxA still works then r+wc

File: public/js/views/page.js (hunk 1 above)
Comment (muffinresearch) [inline public/js/views/page.js]: Ah yes my bad - we're using this elsewhere: "Connecting to Firefox Accounts" so I'll switch to that.

**Comment by muffinresearch** [comment]
Going to wait till after the tag to land this. The reset flow still has a click view too so we'll want to get testing of both bits next week.

**Comment by muffinresearch** [comment]
Looking at the identity code I noticed there's still code that looks at whether the sign-in has come from a click or not. As a result of that we're not going to be able to lose these pages for native FxA. I'll close the related bugs as invalid and open a new one to re-jig the click screen to make it more FxA friendly.

**Pull request closed without merging**